internal/cluster/transition.go

```diff
@@ -562,3 +572,20 @@ func setPendingUserReset(ctx context.Context, c *common.Cluster, db *gorm.DB, ho
 	txSuccess = true
 	return nil
 }
+
+func addExtraParams(log logrus.FieldLogger, cluster *common.Cluster, clusterState string) ([]interface{}, error) {
+	extra := []interface{}{}
+	switch clusterState {
+	case models.HostStatusInstalling:
```

Comment:
why not models.ClusterStatusInstalling ?

Reply:
yes, this is more relevant. Thank you.